webapp/packages/plugin-codemirror6/src/LANG_EXT.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { javascript } from '@codemirror/lang-javascript';
+import { json } from '@codemirror/lang-json';
+import { sql } from '@codemirror/lang-sql';
+import { xml } from '@codemirror/lang-xml';
+
+export const LANG_EXT = {
+  sql,
+  javascript,
+  json,
+  xml,
+};
```

Comment:
here is chunk splitting problem, because every lang is heavy and re exporting them as constants killing tree-shaking
we can use async import here or proper reexports:
`export { javascript as EDITOR_JAVASCRIPT } from '@codemirror/lang-javascript';`